### ISSUE TITLE
Decide whether to use pointer or not for input data

### ISSUE THREAD

**Issue by BenjaminRueth** [FMenhorn/CFDLab#1]
I think we should decide whether to use pointers or not for input data like dx, dy, dt, alpha and so on. In tryout.c this is not solved consistently.

**Comment by FMenhorn** [comment]
yep, good point. Since I am not in Uni tomorrow I ll write here. I think in general its better to not use pointers, since then you do not have to dereference, it simplifies the code (no malloc and so on) and you are in general less error-prone. The reason to use them as pointers is if you want to change them in a function without using the return statement (e.g. if you have more then one return value). Then, however, just give the value as address/reference to the function. That's why I changed these values back to non-pointers. dt could make sense as pointer because it changes during the calculation. But -as you said- for consistency we could also change that as well.
